### PR TITLE
Packaging minutes 2017 11 29

### PR DESCRIPTION
**Pull request by drbenmorgan**
This adds the post for minutes of the Packaging WG meeting of 29/11/17, and also proposes a slight change to the layout of the activities page for the group.

In the latter case, no longer list meeting explicitly, but highlight the Indico site.

### CODE REVIEW

**Comment by drbenmorgan** [comment]
I've checked the Travis log, and the error seems to be a LinkChecker error rather than anything to do with this PR.

**Comment by graeme-a-stewart** [comment]
Yeah, that URL has a known problem, so we just ignore this failure for now. Will do a proper review later this afternoon...

File: organization/_posts/2017-11-29-packaging.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  of package manager and other operations. Containers interface with each other via
+  abstract file/REST/data server/message queuese
+* Good fit in HEP: global distribution of a complex stack that uses significant OS component to a wide range of systems that we don’t manage like the Grid, HPC user laptops.
+* User developed analysis software has different requirements - often a mix between stages inside release, own code that might require specific directory structure.
+  Physics reproducibility is important.
+* Two approaches in ATLAS for images:
+  * "Fat": Completely self-contained, local on disk, no network or local cvmfs install needed. But large (10GB+).
+  * "Thin": Only provide OS, domain software from host install of CVMFS - needs network, and no immediate knowledge of what bits in CVMFS are used.
+* Build both as part of nightlies for offline and analysis releases, distributed via Docker Hub.
+* Work on CVMFS Graph Driver - “Fat” Image, can be used directly, but also deployed over CVMFS. Docker can then just pull the bits its needs.
```
Comment (graeme-a-stewart) [inline organization/_posts/2017-11-29-packaging.md]: its -> it

File: organization/_posts/2017-11-29-packaging.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+* New way of building Athena is quite simple. LCG release is a base, then build AtlasExternals, then Gaudi, then the rest of Athena.
+* Build of Athena bootstraps using AtlasSetup  (sets up gcc, cmake version etc). A few helper scripts used for building whole of Athena, or subsets - output is rpm, from CPack.
+* Nightly rpms are release candidates - copied to EOS. Distribution to CVMFS via Python script. Each nightly build put into date-time tagged directories, under this are the Athena, AthenaExternals, Gaudi. LCG, Externals, tdaq are common over a range of Nightlies
+* Also allow private builds to be distributed over CVMFS
+* Future: looking at ways to save AtlasExternals binary artifacts. Reuse? Looking at Nexus, git LSF. Breaking up large rpms. Include build of LCG
+
+### Discussion
+* Microsoft has released source code for their Git vistual filesystem (GVFS). There is a project on GitHub to port this to Mac and then Linux. Will be a talk on this at CVMFS Workshop in January. Background: Windows source code is very large, so using a Virtual File System to only checkout files when touched.
```
Comment (ktf) [inline organization/_posts/2017-11-29-packaging.md]: Not a project "on Github" by a project by Github and Microsoft, at least as far as I understand it. The whole story at https://arstechnica.com/gadgets/2017/11/microsoft-and-github-team-up-to-take-git-virtual-file-system-to-macos-linux/

**Pull request merged**